### PR TITLE
NHWC Conv2d Bwd weight fp16 ckprofiler and test

### PR DESCRIPTION
**Pull request by ltqin**

### CODE REVIEW

**Review by asroy — APPROVED** [review]

File: example/11_conv2d_bwd_wgt/CMakeLists.txt [hunk 1]
```diff
@@ -1 +1 @@
-add_example_executable(example_conv2d_bwd_wgt_xdl conv2d_bwd_wgt_xdl.cpp)
+add_example_executable(example_conv2d_bwd_weight_xdl conv2d_bwd_wgt_xdl.cpp)
```
Comment (asroy) [inline example/11_conv2d_bwd_wgt/CMakeLists.txt:1]: Please make sure the file name and the executable name are consistent. You can update the file name

**Comment by asroy** [comment]
FP32, bpf16 will be for future PR?

**Comment by asroy** [comment]
@ltqin Please fix file naming and merge conflict

File: example/11_conv2d_bwd_wgt/CMakeLists.txt (hunk 1 above)
Comment (ltqin) [reply example/11_conv2d_bwd_wgt/CMakeLists.txt]: done

**Comment by ltqin** [comment]
> FP32, bpf16 will be for future PR?

how should we do if data is bf16, c matrix  transfer to fp32?

**Pull request merged**